distribution/bin/enso.bat

```diff
@@ -6,5 +6,5 @@ if /I %%A==--dump-graphs (
 set EXTRA_OPTS=%EXTRA_OPTS% -Dgraal.Dump=Truffle:1
 )
 )
-java --module-path %comp-dir% -Dpolyglot.compiler.IterativePartialEscape=true %EXTRA_OPTS% %JAVA_OPTS% -m org.enso.runtime/org.enso.EngineRunnerBootLoader %*
+java --module-path %comp-dir% --add-modules org.jline -Dpolyglot.compiler.IterativePartialEscape=true %EXTRA_OPTS% %JAVA_OPTS% -m org.enso.runtime/org.enso.EngineRunnerBootLoader %*
```

Comment:
It'd be better to avoid these `add-modules` modifications. If we are sure we want to use `org.jline` as a module in enso, then I'd suggest to modify `module-info.java` to `require static org.jline`. Then the standard JVM launcher shall pick the `org.jline` module up, when it is available.

Reply:
Good idea. Let's avoid `--add-modules`.